womtool/src/main/scala/womtool/wom2wdlom/WomToWdlom.scala

```diff
@@ -266,7 +267,8 @@ object WomToWdlom {
               scatterName = a.identifier.localName.value,
               scatterExpression = expression,
               scatterVariableName = a.inputPorts.toList.head.name,
-              graphElements = graph
+              graphElements = graph,
+              srcLoc = None
```

Comment:
Is it intended that you're not forwarding the scatter source locations into the WOM scatters?

Reply:
This is the reverse direction used by the WDL writer. Got me too.